### DISCUSSION TITLE
Customize Header Link Symbol

### DISCUSSION THREAD

**Discussion by kevinzakka** [pradyunsg/furo#580] [Questions] [answered]
Hi @pradyunsg,

Is there a way we can customize the header link symbol?

(read-the-docs)
<img width="68" alt="Screen Shot 2022-11-16 at 3 45 55 PM" src="https://user-images.githubusercontent.com/10518920/202226975-0216d873-732b-4015-ae8f-6060abc9ed54.png">

(furo)
<img width="68" alt="Screen Shot 2022-11-16 at 3 46 40 PM" src="https://user-images.githubusercontent.com/10518920/202227073-b22158de-0df7-46bd-b2a3-a837ee4f07ca.png">

**Comment by pradyunsg** [comment, marked as answer]
Yes! It's a configuration variable provided by Sphinx.

https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_permalinks

Because I enjoyed actaully implementing this and sharing things is fun -- here's the details... Furo actually has [a bit of a hack (IMO)](https://github.com/pradyunsg/furo/blob/b7d8866c3aa1f9f293730c0868a0d52cd6f12f8a/src/furo/__init__.py#L253-L257) to make this `#`, which is a [different hack](https://github.com/readthedocs/sphinx_rtd_theme/blob/f854c890e1445b99a43ac5f10463a5df83739a2e/sphinx_rtd_theme/__init__.py#L65) than what sphinx-rtd-theme uses -- I think Furo's hack is "better" since it allows the user to override the symbol if they want.

In an ideal world though, no one should need hacks to make things work: https://github.com/sphinx-doc/sphinx/issues/10275 -- something that everyone agrees on, with only a PR needed to improve the state of affairs on this front. :)

**Reply by kevinzakka** [reply]
Thank for the details @pradyunsg, appreciate it as always!
